figure_notebooks/SLE_PLIER_figure.Rmd

```diff
@@ -0,0 +1,249 @@
+---
+title: "Tweaking display items: SLE U plot, AUC cut points"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**J. Taroni 2018**
+
+## Set up 
+
+```{r setup}
+knitr::opts_knit$set(root.dir = "..")
+```
+
+```{r}
+library(PLIER)
+`%>%` <- dplyr::`%>%`
+```
+
+```{r}
+plot.dir <- file.path("figure_notebooks", "figures")
+# color palette for PCA plots
+plot.color.pal <- c("#54FF9F", "#43CD80", "#2E8B57", "#006400", "#FF8C00",
+                    "#8B4500", "#000080")
+```
+
+## Read in data
+
+PLIER model
+
+```{r}
+# read in SLE WB PLIER model
+plier.file <- file.path("results", "05", "SLE-WB_PLIER_model.RDS")
+sle.plier <- readRDS(plier.file)
+```
+
+Sample-dataset mapping file
+
+```{r}
+# read sample dataset mapping file
+sd.file <- file.path("data", "sample_info", "sle-wb_sample_dataset_mapping.tsv")
+sd.df <- readr::read_tsv(sd.file)
+```
+
+## Plotting
+
+### `U` plot with selected latent variables
+
+```{r}
+pdf(file.path(plot.dir, "SLE_U_auc0.75_custom_col_index.pdf"))
+PLIER::plotU(sle.plier, auc.cutoff = 0.75, fontsize_row = 7,
+             fontsize_col = 7, indexCol = c(6, 69, 110, 87, 34, 2, 27, 122, 86,
```

Comment:
What do these integers mean? Selecting particular columns?

Reply:
Yes, I've added a note about this in an upcoming commit.